src/plugins/home/server/services/sample_data/usage/usage.ts

```diff
@@ -37,19 +37,23 @@ export function usage(
     logger.warn(`saved objects repository incrementCounter encountered an error: ${err}`);
   };
 
-  const internalRepository = savedObjects.createInternalRepository();
+  const internalRepository = savedObjects.then(so => so.createInternalRepository());
 
   return {
     addInstall: async (dataSet: string) => {
       try {
-        await internalRepository.incrementCounter(SAVED_OBJECT_ID, dataSet, `installCount`);
+        await (await internalRepository).incrementCounter(SAVED_OBJECT_ID, dataSet, `installCount`);
```

Comment:
optional: I'd expect something similar to https://github.com/elastic/kibana/pull/55012/files#diff-b7a00ca0a0cbdb5c3792d6a7dbfd6811R67-R74

Reply:
Missed this one when cleanup up the calls, thanks